src/components/App.js

```diff
@@ -34,7 +38,49 @@ class App extends Component {
     }
 
     console.log(('Starting the app for platform: ' + this.state.os));
+    this.initLinkedIn();
+  }
+
+  // Initialize linkedin tokens
+  initLinkedIn() {
+    let redirecUrl = 'http://localhost';
```

Comment:
redirecUrl -> redirectUrl

and this can't be right. Why are we redirecting anything to localhost for client? Is react native running a server to transcode js files into native code? If that's the case, can we leave a comment what this localhost is doing?